src/core/EdFi.DataManagementService.Core/Handler/UpsertHandler.cs

```diff
@@ -14,6 +14,7 @@
 using Polly;
 using static EdFi.DataManagementService.Core.External.Backend.UpsertResult;
 using static EdFi.DataManagementService.Core.Response.FailureResponse;
+using static EdFi.DataManagementService.Core.Handler.Utility;
```

Comment:
`using EdFi.DataManagementService.Core.External.Backend;` is an unused using in this file